platforms/centos/README.md

```diff
@@ -0,0 +1,28 @@
+# CentOS
+
+[**CentOS**](https://centos.org) is derived from Red Hat Enterprise Linux (RHEL), without requiring licenses to run. The instructions below will also work for RHEL 8 and later.
+
+First, open the **Terminal** application and run the following command to install the EPEL (Extra Packages for Enterprise Linux) repository:
+
+```
+sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+```
+
+> **Note**: This command requires administrator privileges, so it’ll ask for your password.
+
+After the EPEL repository has been added, run the following command to install Swift:
+
+```
+sudo dnf install -y swift-lang
```

Comment:
Add `git` here.

Reply:
Huh, I hadn't seen that git was explicitly required, which of course makes total sense. I will add it to the list of dependencies so that it will be automatically installed starting with 5.3.1 or whatever the next version number will be.